### ISSUE TITLE
Issue with random spawn

### ISSUE THREAD

**Issue by dogyg** [joshuaprince/CoordinateOffset#6]
I applied the min and max values in the random spawn plugin and the player spawns almost at the end of the world, so the player gets stuck outside the worldboarder and dies. Is there any way to fix this?

**Comment by joshuaprince** [comment]
Hi! Can you please provide some more information?

- Could you share a link to the random spawn plugin you are using?
- What changes have you made in the configuration for the random spawn plugin?
- What changes have you made in the CoordinateOffset config.yml?

**Comment by dogyg** [comment]
The link to the random spawn plugin I'm using is https://github.com/Yellowstrawberrys/RandomSpawn and it has the following values
The random spawn configuration has the following values: centerX/Z, maxX/Z, and canSpawnInWater. I set all of the centerX/Z to 0 and the maxX/Z value to 100000.
I tried to change the CoordinateOffset configuration value, but I don't really understand it, so I'm using the default value of Random.
